src/main/java/com/autotune/Autotune.java

```diff
@@ -173,6 +174,9 @@ public static void main(String[] args) {
             startAutotuneNormalMode(context);
         }
 
+        // Check the settings initially while starting
+        SettingsUpdater.initialiseCentralSettings();
```

Comment:
Please give it a better name, CentralSettings doesn't elaborate what these settings are about

Reply:
How about `MainSettings`?